schemas/common/b2b-lead.schema.json

```diff
@@ -0,0 +1,98 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/b2b-lead",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Lead",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "This Data Type is used to capture B2B Lead specific attributes.",
+  "definitions": {
+    "b2b-lead": {
+      "properties": {
+        "xdm:ID": {
+          "title": "Lead Id",
+          "description": "Lead id.",
+          "type": "string"
+        },
+        "xdm:converted": {
+          "title": "Converted Indicator",
+          "description": "Indicates whether the Lead was successfully converted or not.",
+          "type": "boolean"
+        },
+        "xdm:convertedAccountID": {
+          "title": "Converted Account ID",
+          "description": "Account Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:convertedContactID": {
+          "title": "Converted Contact ID",
+          "description": "Contact Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:convertedDate": {
+          "title": "Converted Date",
+          "description": "Date of conversion if the lead was successfully converted.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:convertedOpportunityID": {
+          "title": "Converted Opportunity ID",
+          "description": "Opportunity Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:jigsawContactID": {
+          "title": "Jigsaw Contact Id",
+          "description": "Lead’s Data.com ID if available.",
+          "type": "string"
+        },
+        "xdm:lastTransferDate": {
+          "title": "Last Transfer Date",
+          "description": "Last transfer date.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:unReadByOwner": {
```

Comment:
`readByOwner`

Reply:
This is coming straight from the "Salesforce" entity "Lead". there were lot of flags with "Is" prefix. We just removed the "Is" prefix and retained the name as such.

Reply:
@mjasrotia if this is coming from another schema, then you need to use that schema's namespace.  So instead of `xdm:`, how about making them `sf:`?

Reply:
Initially, we had an intention to create namespaces for Salesforce, MSFT dynamics and Common Data Model but then we decided to have a common set of schemas for industry standard B2B entities and their properties. So, we brought all properties under the same "xdm" namespace. Any customer specific property was created in its own mixin under that customer specific namespace like what we did for "marketo"

Reply:
Added "is" prefix